sap/adt/core.py

```diff
@@ -1,22 +1,23 @@
 """Base ADT functionality module"""
 
 import os
+import urllib
+from abc import ABC, abstractmethod
+from typing import Any, NoReturn, Optional, Union
+from dataclasses import dataclass
 
 import xml.sax
 from xml.sax.handler import ContentHandler
 
 import requests
 from requests.auth import HTTPBasicAuth
+import urllib3
```

Comment:
How is this change related to ADT over RFC?

Reply:
Reverted.